nbs/methods.ipynb

```diff
@@ -13,7 +13,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Could you please remove the outputs? (Edit &gt; Clear All Outputs) (The documentation is generated even if the notebooks don't have it displayed).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Hi, for each notebook I did Cell -> All Output -> Clear, I hope that is the same thing, I don't see the Edit > Clear All Outputs in jupyter notebook